date and time can now be set

DGovi opened the pull request:
closes #23

DGovi committed:
date and time can now be set

adrientremblay commented on soen343house.iml:
```diff
@@ -1,28 +1,12 @@
     </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
-    <orderEntry type="module-library">
```
I need this whole block for intelliJ to be able to find org.json.JSONException  :(

DGovi replied:
very cool ill put it back

DGovi replied:
nvm it gives me a bunch of errors when I add it

adrientremblay commented on soen343house.iml:
```diff
@@ -1,28 +1,12 @@
-      </library>
-    </orderEntry>
-    <orderEntry type="library" name="jfxrt" level="project" />
+    <orderEntry type="library" name="dependencies" level="project" />
```
because of the way i have my libraries set up  name needs to be 'jfxrt' for me.  We should keep this consistent as a group.  Is this break your guy's intelliJ?  @MahadJanjua  @genericalexacc  @jmezaber ?

adrientremblay commented on src/View/Dashboard.fxml:
```diff
@@ -1,44 +1,52 @@
+            <Button mnemonicParsing="false" prefHeight="27.0" prefWidth="136.0" text="Change Temp" />
+                  <HBox prefHeight="18.0" prefWidth="132.0">
+                     <children>
+                        <Label fx:id="dateLabel" prefHeight="17.0" prefWidth="133.0" text="Date is:" />
```
Im getting ...s because of some width issues with the labels.

![image](https://user-images.githubusercontent.com/8109538/96387888-66e30f80-1173-11eb-881d-80f8da594361.png)

DGovi replied:
thanks i missed this!

DGovi committed:
small visual bug fix where date was not displayed fully

adrientremblay commented on src/Controller/DashboardController.java:
```diff
@@ -339,5 +347,24 @@ else if(door == doorsTop.get(0) || doorsTop.size() > 1 && door == doorsTop.get(1
-    
+
+    //shows entered date in label box
+	public void displayDate(javafx.event.ActionEvent actionEvent) {
```
I just realized that these two methods do not update the actual date and time parameters of the simulation class.  I would look at Mahad's createUser() method for inspiration.  Essentially it just calls an associated method within the Simulation class that has all the logic within it.   That returns a string which will be the  result message thats printed to console.  Then the updateDashboard() is the method that actually changes the label values from the values that are in the simulation class.

adrientremblay replied:
yeah so I everything works but as I sad before we should set the data and time parameters for the Simulation class.  Also this logic should be moved inside methods of the simulation class.  Look at the createUser() method or my set-house-location branch (which has a PR open) for inspiration.

DGovi committed:
added javaDoc for all methods and classes

DGovi committed:
changed iml file

adrientremblay commented on src/Controller/DashboardController.java:
```diff
@@ -162,6 +180,10 @@ else if (editUserType.getValue() == null && editUserNewPassword.getText().length
 	}
 
+	/**
+	 * User is deleted from teh simulation and can no longer use
```
"teh"

adrientremblay commented on src/Controller/DashboardController.java:
```diff
@@ -300,7 +334,14 @@ else if(door == doorsTop.get(0) || doorsTop.size() > 1 && door == doorsTop.get(1
-	
+
+	/**
+	 * On a given house layout, draw the windoes at specified location
```
"windoes"

DGovi replied:
bruh ahahahah

adrientremblay reviewed (changes requested):
Everything works and is good to go except that the methods inside DashboardController do not actually update sim (see comment).  Also we might need to add more to the javadocs because I watched the professor's lecture on javadocs and he was like "add more detail then what is visible from the class name" for javadocs.  But as a baseline they're def good.

DGovi committed:
date and time method return as strings

date and time methods in dashboard controller return strings instead of void
also little javadoc typos fixed

DGovi committed:
date and time connected to sim

also removed the date label stuff becuase its already there with the datepicker

adrientremblay committed:
literally added one missing line

adrientremblay committed:
Merge branch 'master' into DateAndTime
